angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-details.html

```diff
@@ -1,120 +1,106 @@
-<div ng-controller="MarketplaceDetailsController as detailsCtrl" class="portlet-frame col-xs-12 col-md-12 portlet-details-page">
-  <div ng-if='loading' class="row details-header no-margin">
-    <loading-gif data-object='obj' data-empty='!loading'></loading-gif>
-  </div>
+<div ng-controller="MarketplaceDetailsController as detailsCtrl" class="portlet-frame portlet-details-page" flex="100">
+    <md-card>
+        <div ng-if='loading' class="row details-header no-margin">
+            <loading-gif data-object='obj' data-empty='!loading'></loading-gif>
+        </div>
+        <div ng-if='error'>
+            <div class="alert alert-danger" role="alert" style='margin: 20px;'>
+                <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+                <span class="sr-only">Error:</span>
```

Comment:
❤️  the attention to accessibility. This did make me go look at [ARIA guidance on alerts](https://www.w3.org/TR/wai-aria-practices/#alert) and [on labeling](https://www.w3.org/TR/wai-aria-practices/#LabeledBy). This "alternative access" approach of having one span for typical browsers and another span for screen readers, I don't think it's bad, might well be the best practice for all I know :) but would also like to  bear in mind approaches that make a shared element "responsive" to different ways of consumption.